package Protos;

import "base.proto";

message SpikeNeuronInfo {
    required double u = 1;
    required double p = 2;
    required uint32 id = 3;
    required uint32 xi = 4;
    required uint32 yi = 5;
    required uint32 colSize = 6;
    required double axonDelay = 7;
    required uint32 num_of_synapses = 8;
    required bool act_function_is_set = 9;
    required bool input_is_set = 10;
    required bool lrule_is_set = 11;
    required bool reinforce_is_set = 12;
    required bool weight_normalization_is_set = 13;
}

